modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -289,6 +331,8 @@ View your coverage report to see your custom connector coverage. You must have a
 . In *Choose Coverage Suite to Display*, add `jacoco-munit.exec` to the list if it's not there already.
 . Look at the coverage percentages to analyze your results.
 
+The coverage percentage reflects the confidence line in which you can state your connector is Java 17 compatible. A key issue to avoid is illegal reflective access in your code. The only ways to detect this are to <<review-your-code>> or through testing. The higher the coverage percentage, the better your chances of catching Java 17 violations. Aiming for at least 80% coverage allows flexibility on simpler code, such as getters and setters, while ensuring thorough testing of all critical business logic.
```

Comment:
`The coverage percentage reflects the confidence line in which you can state your connector is Java 17 compatible.` -> Misleading. THe coverage percent reflects how much of your connector is tested. That percentage does not reflect the JVM used to test, and that percentage needs to be high regardless of the Java version